File: 2220-minimum-bit-flips-to-convert-number/2220-minimum-bit-flips-to-convert-number.py
class Solution:
    def minBitFlips(self, start: int, goal: int) -> int:
        res = 0
        while start != 0 or goal !=0:
            res += (start  & 1) ^ (goal & 1)
            start >>= 1
            goal >>= 1
        return res

File: 2220-minimum-bit-flips-to-convert-number/2220-minimum-bit-flips-to-convert-number.java
class Solution {
    public int minBitFlips(int start, int goal) {
        int res = 0;
        while(start != 0 || goal !=0 ) {
            res += (start & 1) ^ (goal & 1);
            start = start >> 1;
            goal = goal >> 1;
        }
        return res;
    }
}